router/state/manager.go

```diff
@@ -475,8 +475,8 @@ func (sm *ManagerImpl) GetApiSession(token string) *ApiSession {
 		jwtToken, accessClaims, err := sm.ParseJwt(token)
 
 		if err == nil {
-			if !accessClaims.HasAudience(common.ClaimAudienceOpenZiti) {
-				pfxlog.Logger().Errorf("provided a token with invalid audience '%s', expected: %s", accessClaims.Audience, common.ClaimAudienceOpenZiti)
+			if !accessClaims.HasAudience(common.ClaimAudienceOpenZiti) || !accessClaims.HasAudience(common.ClaimLegacyNative) {
```

Comment:
would it make sense at all to log when it's 'native' to let people know they have a problem? or will that sort of thing happen in another stage maybe?

Reply:
also somewhat sad that the code isn't DRY - i take it they are three different structs?

Reply:
>DRY 
How would you change it? I don't see an issue.

> sense at all to log
I don't see why logging would do anything to help at this point in time. The goal is to remove the client id native. I am simply not breaking the test HA environment. The goal is to remove native once the C SDK is updated.

Reply:
After talking, these concerns are addressed when `native` support is removed.